architecture/additional_concepts/authorization.adoc

```diff
@@ -475,53 +479,56 @@ Validates against all ranges.
 . *RunAsAny* - No default provided. Allows any `*supplementalGroups*` to be
 specified.
 
+[[authorization-FSGroup]]
 ==== FSGroup
 
 . *MustRunAs* - Requires at least one range to be specified if not using
 pre-allocated values. Uses the minimum value of the first range as the default.
 Validates against the first ID in the first range.
 . *RunAsAny* - No default provided. Allows any `*fsGroup*` ID to be specified.
 
+[[authorization-controlling-volumes]]
 === Controlling Volumes
 
-The usage of specific volume types can be controlled by setting the `*volumes*` field
-of the SCC.  The allowable values of this field correspond to the volume sources that
-are defined when creating a volume:
-
-.  azureFile
-.  flocker
-.  flexVolume
-.  hostPath
-.  emptyDir
-.  gcePersistentDisk
-.  awsElasticBlockStore
-.  gitRepo
-.  secret
-.  nfs
-.  iscsi
-.  glusterfs
-.  persistentVolumeClaim
-.  rbd
-.  cinder
-.  cephFS
-.  downwardAPI
-.  fc
-.  configMap
-.  *
-
-The recommended minimum set of allowed volumes for new SCCs are configMap, downwardAPI, emptyDir,
-persistentVolumeClaim, and secret.
+The usage of specific volume types can be controlled by setting the `*volumes*`
+field of the SCC. The allowable values of this field correspond to the volume
+sources that are defined when creating a volume:
+
+* *azureFile*
+* *flocker*
+* *flexVolume*
+* *hostPath*
+* *emptyDir*
+* *gcePersistentDisk*
+* *awsElasticBlockStore*
+* *gitRepo*
+* *secret*
+* *nfs*
+* *iscsi*
+* *glusterfs*
+* *persistentVolumeClaim*
+* *rbd*
+* *cinder*
+* *cephFS*
+* *downwardAPI*
+* *fc*
+* *configMap*
+*  ***
```

Comment:
does this render as 3 asterisks via our build process? Or do these need to be wrapped in backticks or escaped somehow?